exonum-java-binding-fakes/src/main/java/com/exonum/binding/fakes/mocks/UserServiceAdapterMockBuilder.java

```diff
@@ -36,7 +36,29 @@
 @SuppressWarnings({"unused", "WeakerAccess"}) // Used in native code
 public final class UserServiceAdapterMockBuilder {
 
-  private final UserServiceAdapter service = mock(UserServiceAdapter.class);
+  private final UserServiceAdapter service;
+
+  /**
+   * Default constructor.
+   */
+  public UserServiceAdapterMockBuilder() {
+    this.service = mock(UserServiceAdapter.class);
+
+    // Some default mocking that could be overridden later
+    init_mocks();
```

Comment:
It is good to set defaults here — but if we do that, then this code and the constants
```rust
        ServiceMockBuilder { exec, builder }
            .id(SERVICE_DEFAULT_ID) <- no longer needed
            .name(SERVICE_DEFAULT_NAME) <- no longer needed
```
must be removed.

Reply:
Well, I just missed that place... Now I'm wondering what is better - just add a record with setting defaults for hashes and rollback the changes for Java part OR remove this and finish the Java part? @dmitry-timofeev WDYT - is the first option is acceptable?

Reply:
I would set defaults in a single place only: either in Java constructor or in Rust ctor. Both options are good.

> just add a record with setting defaults for hashes here and rollback the changes for Java part

If by 'here' you mean in the Rust ctor — it sounds perfectly good to me.

Reply:
If you go with the first — I believe some comments on the Java part may be no longer relevant.